hearth/lib/hearth/http2/request.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require_relative '../http/request'
```

Comment:
These don't need to be required, just require http2 after http in the normal place